config/initializers/default_url_options.rb

```diff
@@ -0,0 +1,8 @@
+hosts = {
+  development: "https://localhost",
+  staging:     "https://#{@app_name}-staging.katalyst.com.au",
+  production:  "https://#{@app_name}-production.katalyst.com.au",
+  test:        "https://example.com",
+}.freeze
+
+Rails.application.routes.default_url_options = { host: hosts[Rails.env.to_sym] }
```

Comment:
This isn't a good long-term strategy. It will require manual changes for non-Katalyst projects.

I think that the staging and production environments should be getting this from their environment (ecs env). Please add a related PR to make that happen if it's not there already.

I think that the default_url_options setting should be in the environment files rather than here.